src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.cs

```diff
@@ -428,6 +428,16 @@ public sealed partial class SqlClientFactory : System.Data.Common.DbProviderFact
         public override System.Data.Common.DbDataAdapter CreateDataAdapter() { throw null; }
         /// <include file='../../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientFactory.xml' path='docs/members[@name="SqlClientFactory"]/CreateParameter/*'/>
         public override System.Data.Common.DbParameter CreateParameter() { throw null; }
+        /// <include file='../../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientFactory.xml' path='docs/members[@name="SqlClientFactory"]/CreateDataSourceEnumerator/*'/>
+        public override System.Data.Common.DbDataSourceEnumerator CreateDataSourceEnumerator() { throw null; }
+#if NET6_0_OR_GREATER
```

Comment:
This isn't required anymore as the lowest supported target framework version is `net6.0`.

Reply:
Yes, understood and agreed - this PR pre-dates the removal of .NET Standard support. I'll bring it up to date for review.